rector.php

```diff
@@ -2,13 +2,22 @@
 
 declare(strict_types=1);
 
+/**
+ * This file is part of the Pushover package.
+ *
+ * (c) Serhiy Lunak <https://github.com/slunak>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 use Rector\Config\RectorConfig;
 
 return RectorConfig::configure()
     ->withPaths([
-        __DIR__ . '/examples',
-        __DIR__ . '/src',
-        __DIR__ . '/tests',
+        __DIR__.'/examples',
+        __DIR__.'/src',
+        __DIR__.'/tests',
     ])
     ->withPhp74Sets()
```

Comment:
Should here still be `->withPhp74Sets()`?

Reply:
Yes because I changed and executed this in the follow up PR